dali/pipeline/operators/op_spec.h

```diff
@@ -426,87 +512,53 @@ inline bool OpSpec::TryGetArgumentImpl(
     auto schema_val = schema.FindDefaultValue(name);
     using VT = const ValueInst<S>;
     if (VT *vt = dynamic_cast<VT *>(schema_val.second)) {
-      result = vt->Get();
+      result = static_cast<T>(vt->Get());
+      return true;
+    }
+  }
+  return false;
+}
+
+
+template <typename T, typename S>
+inline std::vector<T> OpSpec::GetRepeatedArgumentImpl(const string &name) const {
```

Comment:
`GetRepeatedArgument` is now separated from `GetArgument`. This allows this function to be simpler and avoid unnecessary instantiations of exoctic types - namely, `Tensor<std::vector<S>>`.